### ISSUE TITLE
Need to add B-->A in testing?

### ISSUE THREAD

**Issue by John1231983** [arnab39/cycleGAN-PyTorch#1]
Hello, great code. 
I think you should add one more option to map from B to A in testing
https://github.com/arnab39/cycleGAN-PyTorch/blob/ecdc9735e426992056c70e78f3143bb6d538e48c/test.py#L24

Besides, we do not need to shuffle in testing. 
```
    a_test_data = dsets.ImageFolder(dataset_dirs['testA'], transform=transform)
    b_test_data = dsets.ImageFolder(dataset_dirs['testB'], transform=transform)


    a_test_loader = torch.utils.data.DataLoader(a_test_data, batch_size=args.batch_size, shuffle=True, num_workers=4)
    b_test_loader = torch.utils.data.DataLoader(b_test_data, batch_size=args.batch_size, shuffle=True, num_workers=4)
```
You save my life~

**Comment by John1231983** [comment]
Sorry. For the first point, you did it but first and the second row of the image. Sorry for my mistaken

**Comment by arnab39** [comment]
Hi John.
Thanks for your suggestion. Yeah, you are right shuffling is irrelevant in testing. I just took random images and tested my trained model.

**Comment by John1231983** [comment]
Thanks. In the cycle GAN, they did two experiments:
1. Patch GAN
2. Pair images

Have you consider it in your repo? I think your repo run very smooth. Thanks
